saleor/graphql/account/types.py

```diff
@@ -475,25 +475,29 @@ def _resolve_orders(orders):
         return OrdersByUserLoader(info.context).load(root.id).then(_resolve_orders)
 
     @staticmethod
-    def resolve_avatar(root: models.User, info: ResolveInfo, size=None, format=None):
+    def resolve_avatar(
+        root: models.User, info: ResolveInfo, size: Optional[int] = None, format=None
+    ):
         if not root.avatar:
             return
 
-        if not size:
+        if size is None:
+            size = 4096
```

Comment:
Maybe we could store this value in settings(e.g. `DEFAULT_THUMBNAIL_SIZE`)?

Reply:
I can turn it into a constant, but I don't think it belongs in settings. We don't want users to change it.